caffe2/CMakeLists.txt

```diff
@@ -273,9 +273,8 @@ if(USE_ROCM)
   target_include_directories(caffe2_hip PRIVATE ${Caffe2_HIP_INCLUDES})
   target_include_directories(caffe2_hip INTERFACE $<INSTALL_INTERFACE:include>)
 
-  IF(BUILD_ATEN)
-    aten_set_target_props(caffe2_hip)
-  ENDIF()
+  # Set standard properties on the target
```

Comment:
can this now be done unconditionally? @petrex please comment here

Reply:
@iotamudelta This should actually have always been done unconditionally. Notice that CUDA builds have it unconditional here: https://github.com/pytorch/pytorch/blob/master/caffe2/CMakeLists.txt#L249